Sources/XCTest/Public/XCAbstractTest.swift

```diff
@@ -52,6 +52,11 @@ open class XCTest {
         perform(testRun!)
     }
 
+    /// Async setup method called before the invocation of `setUpWithError` for each test method in the class.
```

Comment:
This hunk, and the hunk below that starts on 78, are the changes that this PR adds which #326 did not contain.